src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+{
+  "FSC.Title": "Simple Calendar",
+  "FSC.Compact": "Compact",
+  "FSC.Full": "Complet",
+  "FSC.Help": "Aide",
+  "FSC.One": "1",
+  "FSC.Five": "5",
+  "FSC.Thirty": "30",
+  "FSC.OrdinalSuffix.st": "er",
+  "FSC.OrdinalSuffix.nd": "e",
+  "FSC.OrdinalSuffix.rd": "e",
+  "FSC.OrdinalSuffix.th": "e",
+  "FSC.ButtonTitle": "Calendrier",
+  "FSC.AllDay": "Toute la journée",
+  "FSC.Day": "Jour",
+  "FSC.Days": "Jours",
+  "FSC.Month": "Mois",
+  "FSC.Year": "Année",
+  "FSC.Hour": "Heure",
+  "FSC.HourShorthand": "hr",
+  "FSC.Minute": "Minute",
+  "FSC.MinuteShorthand": "min",
+  "FSC.Second": "Seconde",
+  "FSC.SecondShorthand": "sec",
+  "FSC.Forward": "Avancer",
+  "FSC.Back": "Reculer",
+  "FSC.Clear": "Effacer",
+  "FSC.Random": "Aléatoire",
+  "FSC.ShowAdvanced": "Afficher Avancé",
+  "FSC.HideAdvanced": "Masquer Avancé",
+  "FSC.DateControls": "Contrôles de date",
+  "FSC.DateControlsHelp": "Utilisez ces commandes pour modifier la date actuelle dans votre jeu.",
+  "FSC.NoDateControls": "Simple Calendar a été configuré pour laisser un autre module contrôler la date et l'heure. Veuillez utiliser ce module pour mettre à jour l'heure ou modifier la configuration de Simple Calendars.",
+  "FSC.SetCurrentDate": "Définir la date actuelle",
+  "FSC.SetCurrentDateDialog.Title": "Confirmer le changement de date actuelle",
+  "FSC.SetCurrentDateDialog.Content": "Vous avez choisi de changer la date en un jour qui n'est actuellement pas visible !<br/>Voulez-vous vraiment changer la date du jour en <strong>{DATE}</strong> ?",
+  "FSC.Apply": "Appliquer",
+  "FSC.AdvancedOptions": "Options Avancées",
+  "FSC.Today": "Ajourd'hui",
+  "FSC.Configure": "Configurer",
+  "FSC.Import": "Importer",
+  "FSC.Export": "Exporter",
+  "FSC.By": "Par",
+  "FSC.Cancel": "Annuler",
+  "FSC.Edit": "Éditer",
+  "FSC.Delete": "Supprimer",
+  "FSC.DeleteConfirm": "Confirmer Suppression",
+  "FSC.DeleteConfirmText": "Vous voulez vraiment supprimer cette note?",
+  "FSC.OverwriteConfirm": "Écraser Existant",
```

Comment:
I would normally suggest "Écraser les valeurs" (Overwrite values) as a confirmation message, but I think this would be too big for the button. Maybe just "Écraser" (Overwrite)?